reference/interbit-cli/config/README.adoc

```diff
@@ -0,0 +1,62 @@
+= The Interbit Configuration File
+
+The interbit configuration file contains instructions for the
+link:../build.adoc[build] script. It details how to read and bundle your
+covenants and deploy them to a network. The build script will consume a
+configuration and file and resolve the variables inside of it to
+generate a manifest file used to deploy your network.
+
+It is a Javascript file and can be altered at any time.
+
```

Comment:
@nicholeous The "can be altered at any time" sentence was included in the original config topic. I kept it in, as it could be important, but as David mentioned it is unclear. Can you clarify the intent of that sentence?

Reply:
The user may alter this file at any time (contrary to the manifest which will be corrupted and is for the build script to update)